x-pack/plugins/security_solution/public/flyout/index.tsx

```diff
@@ -67,42 +63,20 @@ const expandableFlyoutDocumentsPanels: ExpandableFlyoutProps['registeredPanels']
   },
 ];
 
-const OuterProviders: FC = ({ children }) => {
-  return <SecuritySolutionFlyoutUrlSyncProvider>{children}</SecuritySolutionFlyoutUrlSyncProvider>;
-};
-
-const InnerProviders: FC = ({ children }) => {
-  const [flyoutRef, handleFlyoutChangedOrClosed] = useSecurityFlyoutUrlSync();
-
-  return (
-    <ExpandableFlyoutProvider
-      onChanges={handleFlyoutChangedOrClosed}
-      onClosePanels={handleFlyoutChangedOrClosed}
-      ref={flyoutRef}
-    >
-      {children}
-    </ExpandableFlyoutProvider>
-  );
-};
-
 export const SecuritySolutionFlyoutContextProvider: FC = ({ children }) => (
-  <OuterProviders>
-    <InnerProviders>{children}</InnerProviders>
-  </OuterProviders>
+  <ExpandableFlyoutProvider>{children}</ExpandableFlyoutProvider>
 );
 
 SecuritySolutionFlyoutContextProvider.displayName = 'SecuritySolutionFlyoutContextProvider';
 
-export const SecuritySolutionFlyout = memo(() => {
-  const [_flyoutRef, handleFlyoutChangedOrClosed] = useSecurityFlyoutUrlSync();
+const handleFlyoutChangedOrClosed = () => {};
```

Comment:
we probably can remove this entirely, I believe the `handleOnFlyoutClosed` below is optional